Makefile.am

```diff
@@ -1,8 +1,10 @@
+ACLOCAL_AMFLAGS = -I m4
+
 SUBDIRS = sources sources/glue doc
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = webkitgtk-sharp-3.0.pc
 
-EXTRA_DIST = webkitgtk-sharp-3.0.pc.in out/webkitgtk-sharp.dll.config
```

Comment:
@andrewshadura Things look fine, yeah, it's just this and the two other changes that I commented on below that look odd... they're all dllmap related, so we can merge without them and then once the debian package is done you can check if everything is working ok and all the mappings are good or if we need to tweak something. Does that work for you?